Constitution.md

```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
+2. Attend Community Assemblies.
+3. Guides the council on advocacy for underrepresented communities.
+4. Elected by the student body in the CSSA elections.
+5. If not filled by an elected executive in the CSSA elections, An elected executive of an UMSU recognized underrepresented community within the Department of Computer Science may be appointed as the Director of Advocacy for the CSSA. Eg: The Women in Computer Science Council.
+6. Ensure that the CSSA lounge is a safe and inclusive space.
+7. Lead the Community Advocates commitee with up to and including committee members for each community recogized under UMSU.
+8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
 
 ### All Executives
 
-1. Report to committee regarding their activities.
+1. Report to council regarding their activities.
 2. Participate in voting on club related activities.
 3. Participate and assist with Group activities.
-4. Be responsible for any specific tasks delegated to them by general resolution of Council or the president.
-5. Be familiar with these bylaws and the Robert’s Rules of Order and not act outside the jurisdiction of their portfolio.
-6. Report to the Council concerning their activities.
-7. Declare any personal or financial interest in any expenditure of the group funds.
-8. Comply with all regulations set out by the Campus Police of the Dean’s Office, specifically those concerning smoking and drinking on campus.
-9. Prepare, in cooperation with other executives of the same position when applicable, an orientation package for the subsequent person to fill each position.
-10. Attend all regular meetings.
+4. Be responsible for any specific tasks delegated to them by general resolution of Council.
+5. Declare any conflicts of interest.
+6. Declare any personal or financial interest in any expenditure of the group funds.
+7. Comply with all regulations set out by the Campus Police of the Dean’s Office, specifically those concerning smoking and drinking on campus.
+8. Promote Council cohesion and a respectful atmosphere abiding by the University of Manitoba Respectful Workplace and Learning Environment Policy.
+9. Refrain from drug/alcohol use or sexual activity at any CSSA event or in the CSSA lounge. Alcohol consumption is only permitted if the venue of the event distributes alcohol.
+10. Not allow any persons who are not Members of Council to use CSSA Office space in the absence of a Member of Council.
+11. Prepare, in cooperation with other executives of the same position when applicable, an orientation package for the subsequent person to fill each position.
+12. Attend all regular meetings.
+13. Put fair effort into acquiring a proxy, in the event a voting member is unable to attend a Council meeting.
 
 ## Article 2c: Executive Officer Dismissal Protocol
```

Comment:
keep in mind of what we put in the duties in article 2b. Because technically if we don't follow it then they should go into probation.